lib/main_common.pm

```diff
@@ -225,7 +225,7 @@ sub kdestep_is_applicable {
 }
 
 sub consolestep_is_applicable {
-    return !get_var("INSTALLONLY") && !get_var("DUALBOOT") && !get_var("RESCUECD");
+    return !get_var("INSTALLONLY") && !get_var("DUALBOOT") && !get_var("RESCUECD") && !check_var('FLAVOR', 'Gnome-Live');
```

Comment:
It's a bunch of waste-of time to execute all those console tests for the GNOME Next media; nobody is going to be interested in touching those failures; so why test them?